docs/reference/async-processing.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Async Processing
+---
+import {Icons} from '/src/components/icons'
+
+:::info
+
+This page covers the Responsive Async Processor: its benefits, its limitations, how to use it, and how to tune it.
+
+The Async Processor is available with Responsive client **v0.24.0** onward.
```

Comment:
I think we should say 0.25 -- I'm just waiting on one small fix for something the fault injection caught in soak and then I'm going to cut a new release, so it should be out before these docs go live